data/json/monstergroups/military.json

```
@@ -89,7 +92,8 @@
       { "monster": "mon_zombie_milbase_personnel", "weight": 50 },
       { "monster": "mon_zombie_military_pilot", "weight": 5 },
       { "monster": "mon_zombie_flamer", "weight": 1, "cost_multiplier": 30 },
-      { "monster": "mon_feral_soldier", "weight": 75 }
+      { "monster": "mon_feral_soldier", "weight": 75 },
+      { "monster": "mon_feral_soldier_flamer", "weight": 45 }
```

```suggestion
      { "monster": "mon_feral_soldier_flamer", "weight": 1 }
```
Matching zombie relationships